tests/utils/test_pointcloud_io.py

```diff
@@ -1,5 +1,6 @@
 import os
 
+import pypose as pp
```

Comment:
Don’t import this library, just generate the results and hardcode the resulting tensor values in the test